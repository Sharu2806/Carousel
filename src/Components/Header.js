import React from 'react';

const Header = () => {
  return (
    <header tabIndex="0" aria-label="Carousel Test Header" className="header">
      Carousel Test
    </header>
  );
}


export default Header;
