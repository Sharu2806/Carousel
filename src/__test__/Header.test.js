import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Components/Header';

test('Match Screenshot', () => {
  const Component = renderer.create(<Header />);
  expect(Component.toJSON()).toMatchSnapshot();
});
