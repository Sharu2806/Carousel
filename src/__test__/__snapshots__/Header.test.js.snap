// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match Screenshot 1`] = `
<header
  aria-label="Carousel Test Header"
  className="header"
  tabIndex="0"
>
  Carousel Test
</header>
`;
